string name
string[] parents
string[] children
string[] values
float64[] cpt # size(values) * permutation(parents * parents.values)
